crypto/s2n_ecdsa.c

```diff
@@ -30,6 +30,14 @@
 #include "crypto/s2n_openssl.h"
 #include "crypto/s2n_pkey.h"
 
+int s2n_ecdsa_signature_size(const struct s2n_pkey *pkey)
```

Comment:
Would it be worth making this even more specific and seeing s2n_ecdsa_der_signature_size() ?